html_utils/utils.py

```diff
@@ -0,0 +1,21 @@
+from bs4 import BeautifulSoup
+
+def get_title(soup: BeautifulSoup):
+    # return the title of the page of empty string if not found
```

Comment:
use `"""` instead of `#` comment styles when documenting functions. You can also check out this style guide: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html